cloud/deploys/bin/azure/key-vault-setup

```diff
@@ -37,7 +37,8 @@ echo "Generating and setting secrets"
 azure::add_generated_secrets \
   "${vault_name}" \
   "postgres-password" \
-  "app-secret-key"
+  "app-secret-key" \
+  "saml-keystore-pass"
```

Comment:
suggestion (blocking): the functions in this file are fairly generic, please move them or this file to cloud/azure/bin/lib/*.sh